doc_admin/AdminManual.adoc

```diff
@@ -458,55 +525,57 @@ Open Hospital has a folder tree hierarchy which is shown here, where *_<version>
+Depending on the chosen operating system and architecture, the needed external applications and libraries (like MySQL/MariaDB and Java JRE) will be present also:
 
 ** _zulu8.58.0.13-ca-fx-jdk8.0.312-<operating_system>_<arch>/ - Java Virtual Machine
 ** _mariadb-<version>-<arch>/ or _mysql-<version>-<arch>-<build>/_ - MySQL Server or MariaDB Server
 
 <<<
 
-[#config-files]
-== 5 Configuration Files
+[#configuration]
+== 3 Configuration
 
-Customization of Open Hospital depends on settings set in text files called “properties files”. Open Hospital comes with 8 main properties files:
+Configuration of the local Open Hospital instance (CLIENT and PORTABLE mode) is set in the configuration files called “properties files”. The following properties files are placed in the **oh/rsc** subfolder:
 
-* *oh/rsc/*
-** *database.properties* – for Open Hospital database connection
-** *dicom.properties* – for the DICOM viewer module
+** *database.properties* – Open Hospital database connection
+** *dicom.properties* – DICOM viewer module options
 ** *examination.properties* – contains the settings for the ‘examination’ module
-** *log4j.properties* – for the logging system
-** *settings.properties* – for configuring Open Hospital options
-** *sms.properties* – for the SMS Manager module
-** *txtPrinter.properties*– for the text printing system
-** *xmpp.properties* – for the Xmpp Server
+** *log4j.properties* – Logging system and paths
+** *settings.properties* – Open Hospital options and settings
+** *sms.properties* – SMS Manager module
+** *txtPrinter.properties* – Text printing system
+** *xmpp.properties* – Xmpp Server configuration
 
 Furthermore, additional configurations files (not meant to be modified by the user) are:
 
 * *resolutions.xml* –  stores locally the information about webcams
 * *version.properties* – contains the current Open Hospital version
 
-Every configuration file works only [.underline]#locally# and is meant to manage  [.underline]#only a single instance# of Open Hospital application.
+The properties files are related only to the local instance of Open Hospital; different instances may use specific OH configuration with custom properties files.
+The properties files can be generated automatically from the packaged _.dist_ files by editing the configuration options available in the startup scripts **oh.sh / oh.ps1** and launching OH. The properties files can also be modified manually using a text editor.
 
-This means that clients running Open Hospital may run differently according to their configuration files.
+NOTE: Any change to these files requires an application restart to apply the modified settings.
 
-NOTE: Any change to these files requires an application restart to apply the settings.
+NOTE: The properties files are automatically generated/overwritten at any Open Hospital startup, unless MANUAL_CONFIG option is set to "on" in the **oh.sh / oh.ps1** scripts.
```

Comment:
Inconsistent highlighting of the line oh.sh / oh.ps1; sometime plain text and other times bolded.

Reply:
Done.